api/src/main/resources/messages.properties

```diff
@@ -105,3 +105,17 @@ referenceapplication.configuremetadata=Configure Metadata
 
 referenceapplication.formentryapp.manageforms.label=Manage Forms
 
+general.auditInfo=Audit Info
```

Comment:
Some of these might be already in core

Reply:
I couldn't find these in the Core. Let's see here : https://github.com/search?p=2&q=org%3Aopenmrs+general.auditInfo&type=Code

Am I missing anything here?

Reply:
Most of these are already defined in the platform

Reply:
Yes, got it. So I just added required once here